core/templates/services/html-length.service.ts

```diff
@@ -105,6 +125,11 @@ export class HtmlLengthService {
 
     let totalWeight = 0;
     for (let tag of Array.from(dom.body.children)) {
+    /**
+      * Guarding against null due to special cases as
```

Comment:
Guarding against tag.textContent === null, which can arise in special cases as explained in ...

Reply:
Done!